app/controllers/licensees_controller.rb

```diff
@@ -1,56 +1,66 @@
 # frozen_string_literal: true
 
 class LicenseesController < ApplicationController
+
   before_action :authenticate_user!
   before_action :authorize_admin, only: %i[new edit]
+    
+    def new
+        @licensee = Licensee.new
+    end
```

Comment:
[Style]
The indentation is off here. Each indentation should be 2 spaces, not four. You can change the default for this on your text editor.